client/src/components/LogOutButton.jsx

```diff
@@ -0,0 +1,47 @@
+import { Button, useToast } from '@chakra-ui/react';
+import React, { useContext } from 'react';
+import { useCookies } from 'react-cookie';
+import { useHistory } from 'react-router-dom';
+import { RootStoreContext } from '../stores/stores';
+import { makeRequest } from '../utils/api';
+
+const LogOutButton = () => {
+  const history = useHistory();
+  const { uiStore } = useContext(RootStoreContext);
+  const [, setCookie] = useCookies(['userLoggedIn']);
+  const toast = useToast();
+
+  const handleLogOut = async () => {
+    try {
+      await makeRequest('post', 'user/logout');
+      uiStore.userLogOut();
+      setCookie('userLoggedIn', false, { path: '/' });
+      history.push('/');
+      toast({
+        position: 'top',
+        title: 'Logged out',
+        description: 'You have logged out successfully',
+        status: 'success',
+        duration: 2000,
+        isClosable: true,
+      });
+    } catch {
+      toast({
+        position: 'top',
+        title: 'An error occurred.',
+        description: 'Unable to log out',
```

Comment:
I feel like we should handle this better, If you clear your cookies and try to logout, we should still logout. I think we should check the status code on the error, if its a 403 then the user should still be logged out

Reply:
or we had the /user/logout route in the non authenticated paths on the server

Reply:
You're right, I just pushed a commit to check for 403 in the error. let me know what you think